polls/templates/polls/auth_token.html

```diff
@@ -21,24 +21,68 @@
         <h3>API Token</h3>
         <code>{{ token }}</code>
       </div>
-      <p class="mt-3">
+
+      <h4 class="mt-3">Admin API</h4>
+      <p>
         Use following code snippet to make request to admin api endpoint.
       </p>
       <pre>
         <code>
 import requests
 headers = {
-    "Authorization": "Token {{ token.key }}"
+    "Authorization":admin_api_url "Token {{ token.key }}"
     }
 url = "{{ admin_api_url }}"
 response = requests.get(url, headers=headers)
 print(response.json())
 print(response.status_code)
         </code>
       </pre>
+
+      <div class="mt-3">
+        <h4>Annotation API</h4>
+        <p>Fetches all entries of annotation table</p>
+        <pre>
+          <code>
+import requests
+
+headers = {
+    "Authorization": "Token {{ token.key }}"
+    }
+annotation_api = "{{ annotation_api }}"
+response = requests.get(annotation_api, headers=headers)
+# print(response.json())
+# print(response.status_code)
```

Comment:
Please Remove the unnecessary comments to maintain the code quality.

Reply:
Sure, that's reasonable but it's kept for test cases. It would be cleaned later on